architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +629,30 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
 
+Sharding allows the operator to define multiple router groups. Each
+router in the group serves only a subset of traffic (sharding).
```

Comment:
This is not necessarily true. What we do is really partition the routes (via namespace/selectors) to be served by different router instances. And we could easily partition the routes in such a manner that the routers service overlapping sets (ala namespaces `[A-J]*`, `[K-T]*` and the last one is overlapping with the previous `[Q-Z]*`).  Traditional sharding/distribution in such as case becomes a special case of partitioning with no overlapping sets.